test/stripe/stripe_object_test.rb

```diff
@@ -283,7 +283,7 @@ class StripeObjectTest < Test::Unit::TestCase
       e = assert_raises ArgumentError do
         obj.foo = ""
       end
-      assert_match /\(object\).foo = nil/, e.message
+      assert_match %r{\(object\).foo = nil}, e.message
```

Comment:
This is to fix a warning caused by the first argument of `assert_match` being ambiguous with a forward slash being on used. I'm not 100% sure _what_ exactly makes it ambiguous here. An alternative solution is probably to use parenthesis with the method call (i.e. `()`).